src/middleware/index.js

```diff
@@ -1,14 +1,11 @@
-const express = require('@feathersjs/express');
 const notFound = require('./not-found-handler');
-const logger = require('./logger');
+const addLogger = require('./logger');
 
-module.exports = function () {
+module.exports = function (app) {
 	// Add your custom middleware here. Remember, that
 	// just like Express the order matters, so error
 	// handling middleware should go last.
-	const app = this;
 
-	app.use(notFound());
-	app.use(logger(app));
-	app.use(express.errorHandler());
+	app.configure(addLogger); 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4527512)